README.md

```diff
@@ -35,23 +75,21 @@ The following are a list of Ansible playbooks
 
 ### Prerequisites
 
-A client environment with the appropriate entitlements to Red Hat Enterprise 7 and OpenShift Enterprise 3 is required to provision new OpenShift environments. A [host builder docker image](../docker/openshift-host-builder) is available to provide an execution environment. 
+A client environment with the appropriate entitlements to Red Hat Enterprise 7 and OpenShift Enterprise 3 is required to provision new OpenShift environments. A [host builder docker image](../docker/openshift-host-builder) is available to provide an execution environment.
 
 ### Configuring the Inventory
 
 The current supported method of provisioning new OpenShift environments is to utilize OpenStack. The `openstack_key_name` in the `ose3` inventory file at the root of this folder must have the SSH key name specified in order for Ansible to be able to communicate with the newly provisioned machines
```

Comment:
Should include a note about Heat and that the OSP env need heat enabled (and potentially which heat group the user needs to be part of for the tenant - i.e.: `heat_stack_owner`)?